### ISSUE TITLE
Feature Request - Focus Device/Device Group Within pools.config.txt

### ISSUE THREAD

**Issue by rpringst** [RainbowMiner/RainbowMiner#1916]
Similar to the Algorithm and ExcludeAlgorithm fields, is it possible to add a means to use only one type of device, or specific device(s), on a given pool? This would allow for example, one to have their CPU to mine on zpool, and GPU on zergpool, but to also algo hop between all algos available for the filtered device on the given pool. Hopefully this request makes sense!

**Comment by RainbowMiner** [comment]
Yes, that totally makes sense - it's a bit difficult to handle. I'll think about it.
